src/Avalonia.Visuals/Rect.cs

```diff
@@ -421,11 +421,48 @@ public Rect Translate(Vector offset)
         }
 
         /// <summary>
-        /// Gets the union of two rectangles.
-        /// </summary>
-        /// <param name="rect">The other rectangle.</param>
-        /// <returns>The union.</returns>
-        public Rect Union(Rect rect)
+		/// Normalizes the rectangle so both the <see cref="Width"/> and <see cref="Height"/> are positive, without changing the location of the rectangle
```

Comment:
Nit: long line, needs scroll in github file view: https://github.com/AvaloniaUI/Avalonia/blob/b5db73186f8c0181daa29ff7f1aa3870cf2baa12/src/Avalonia.Visuals/Rect.cs#L424

Reply:
done